.github/workflows/maven-release.yml

```
@@ -49,9 +49,13 @@ jobs:
         else
           echo "auto_release=false" >> $GITHUB_ENV
         fi
+    - name: Run tests
+      run: ./mvnw -B package
     - name: Release With Maven
       run: |
         ./mvnw -B -U \
+          -DskipTests \
+          -pl '!java-to-zod-maven-plugin-test' \
```

Additional exclusions can be added like this:
```suggestion
          -pl '!java-to-zod-maven-plugin-test,!other-module' \
```